docs/index.asciidoc

```diff
@@ -64,6 +65,20 @@ input plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+  * Value type is <<string,string>>
+  * Supported values are:
+    ** `disabled`: does not use ECS-compatible field names (e.g. `priority` for syslog priority)
+    ** `v1`: uses fields that are compatible with Elastic Common Schema (e.g. `[log][syslog][priority]`)
```

Comment:
Please replace both instances of  "e.g." with "such as," "for example," or similar for Elastic style